.sdkmanrc

```diff
@@ -1,3 +1,3 @@
 # Enable auto-env through the sdkman_auto_env config
 # Add key=value pairs of SDKs to use below
-java=22.2.r11-grl
+java=17.0.11-graal
```

Comment:
Why did we need this change?

Reply:
- I initialized a new Ubuntu WSL distribution instance to test the current project, and it turns out that the project requires both GraalVM CE For JDK11 and GraalVM CE For JDK22 to be installed via SDKMAN! to run unit tests. I will roll back this change.